tests/Unit/MessageBuilder/MessageBuilderTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EdifactParser\Tests\Unit\MessageBuilder;
+
+use EdifactParser\MessageDataBuilder\Builder as MessageDataBuilder;
+use EdifactParser\Segments\DTMDateTimePeriod;
+use EdifactParser\Segments\LINLineItem;
+use EdifactParser\Segments\QTYQuantity;
+use EdifactParser\Segments\RFFReference;
+use EdifactParser\Segments\SegmentInterface;
+use EdifactParser\Segments\UNSSectionControl;
+use PHPUnit\Framework\TestCase;
+
+class MessageBuilderTest extends TestCase
+{
+    private SegmentInterface $referenceSegment;
+    private SegmentInterface $dateTimeSegment;
+    private SegmentInterface $otherDateTimeSegment;
+    private SegmentInterface $lineSegment;
+    private SegmentInterface $quantitySegment;
+    private SegmentInterface $otherLineSegment;
+    private SegmentInterface $otherQuantitySegment;
+    private SegmentInterface $separatorBetweenDetailsAndSummarySegment;
+
+    public function __construct(?string $name = null, array $data = [], int|string $dataName = '')
```

Comment:
Have you tried using the `setUp()` instead of the `__construct()`?

Reply:
I'll give it a try